docs/pages/kubernetes-access/helm/reference/teleport-cluster.mdx

```diff
@@ -741,6 +795,62 @@ Sets the `Group` of `Issuer` to be used when issuing certificates with `cert-man
   </TabItem>
 </Tabs>
 
+### `highAvailability.certManager.issuerSpec`
+
+| Type | Default value | Can be used in `custom` mode? | `teleport.yaml` equivalent |
+| - | - | - | - |
+| `object` | `{}` | ❌ | None |
+
+Spec of `Issuer` or `ClusterIssuer`.
```

Comment:
I don't see a way to toggle from `Issuer` to `ClusterIssuer` currently?

Reply:
You can do it by [`highAvailability.certManager.issuerKind`](https://goteleport.com/docs/kubernetes-access/helm/reference/teleport-cluster/#highavailabilitycertmanagerissuerkind) option.

Reply:
You can do it by [`highAvailability.certManager.issuerKind`](https://goteleport.com/docs/kubernetes-access/helm/reference/teleport-cluster/#highavailabilitycertmanagerissuerkind) option.